mariadb.env.example

```diff
@@ -0,0 +1,9 @@
+MARIADB_AUTO_UPGRADE=true
+MARIADB_ROOT_HOST=localhost
```

Comment:
Shouldn't this be `%` instead of `localhost` to allow connections from database GUIs?

Reply:
Sure should! Changed in 6fd33ab